tests/js/reload-tests.js

```diff
@@ -65,16 +66,16 @@ fluid.defaults("gpii.pouch.tests.reload.caseHolder", {
     }
 });
 
-fluid.defaults("gpii.pouch.tests.reload.environment", {
-    gradeNames: ["gpii.pouch.tests.environment"],
+fluid.defaults("gpii.tests.pouch.reload.environment", {
+    gradeNames: ["gpii.test.pouch.environment"],
     port:       6792,
```

Comment:
This has already been addressed in our call (by moving the old defaults to the test harness launcher).